app/src/ui/app.tsx

```diff
@@ -364,8 +366,40 @@ export class App extends React.Component<IAppProps, IAppState> {
       this.showPopup({ type: PopupType.MoveToApplicationsFolder })
     }
 
+    this.setOnOpenBanner()
+  }
+
+  private onOpenAccessibilitySettings = () => {
+    this.props.dispatcher.showPopup({
+      type: PopupType.Preferences,
+      initialSelectedTab: PreferencesTab.Accessibility,
+    })
+  }
+
+  /**
+   * This method sets the app banner on opening the app. The last banner set in
+   * this method will be the one shown as only one banner is shown at a time.
+   * The only exception is the update available banner is always
+   * prioritized over other banners.
+   *
+   * Priority:
+   * 1. OS Not Supported by Electron
+   * 2. Accessibility Settings Banner
+   * 3. Thank you banner
+   */
+  private setOnOpenBanner() {
     this.checkIfThankYouIsInOrder()
 
+    if (
+      enableDiffCheckMarksAndLinkUnderlines() &&
+      getBoolean(accessibilityBannerDismissed) !== true
+    ) {
+      this.setBanner({
+        type: BannerType.AccessibilitySettingsBanner,
+        onOpenAccessibilitySettings: this.onOpenAccessibilitySettings,
+      })
```

Comment:
Should we return here? Not that it matters much now but if this becomes a pattern where we have even more banners that could show on launch we would end up showing one banner only to show another a moment later. Seems better that we're explicit about the priority amongst these launch banners.

Reply:
That was my first instinct, but then I followed the pattern set with the "no longer supported by electron" banner which overwrote the thank you banner if existed.  I am happy to switch to that. I need to not only add returns statements in, but change the order so that the "electron banner" is prioritized first and thank you banner last